environments/kubernetes/manifests/thanos-compactor-statefulSet.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  labels:
+    app.kubernetes.io/name: thanos-compactor
+  name: thanos-compactor
+  namespace: monitoring
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: thanos-compactor
+  serviceName: thanos-compactor
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: thanos-compactor
+    spec:
+      containers:
+      - args:
+        - compact
+        - --wait
+        - --retention.resolution-raw=16d
+        - --retention.resolution-5m=42d
+        - --retention.resolution-1h=180d
+        - --objstore.config=$(OBJSTORE_CONFIG)
+        env:
+        - name: OBJSTORE_CONFIG
+          valueFrom:
+            secretKeyRef:
+              key: thanos.yaml
+              name: thanos-objectstorage
+        image: quay.io/thanos/thanos:v0.6.1
+        name: thanos-compactor
+        ports:
+        - containerPort: 10902
+          name: http
+        resources:
+          limits:
+            cpu: 500m
+            memory: 2Gi
```

Comment:
so here is 2GI but the jsonnet specified 5GB :thinking: 

>   { name: 'THANOS_COMPACTOR_MEMORY_LIMIT', value: '5Gi' },

Reply:
That's because this file `environments/kubernetes/manifests/thanos-compactor-statefulSet.yaml` is a generic combined Kubernetes deployment for Observatorium, where's the `{ name: 'THANOS_COMPACTOR_MEMORY_LIMIT', value: '5Gi' },` is for OpenShift and more closely to what we actually want on our deployments already. Although we still override these values on a per cluster basis later.

Reply:
Thanks!